packages/contracts-bedrock/medusa.json

```
@@ -0,0 +1,82 @@
+{
+	"fuzzing": {
+		"workers": 10,
+		"workerResetLimit": 50,
+		"timeout": 0,
+		"testLimit": 500,
+		"callSequenceLength": 100,
+		"corpusDirectory": "",
+		"coverageEnabled": true,
+		"targetContracts": ["SuperchainERC20FactoryFuzz"],
+		"targetContractsBalances": [],
+		"constructorArgs": {},
+		"deployerAddress": "0x30000",
+		"senderAddresses": [
+			"0x10000",
+			"0x20000",
+			"0x30000"
+		],
+		"blockNumberDelayMax": 60480,
+		"blockTimestampDelayMax": 604800,
+		"blockGasLimit": 125000000,
```

on Optimism, the block gas limit is 30m
```suggestion
		"blockGasLimit": 30000000,
```
And also, 20m is the limit of the gas that can be deposited from L1 -> L2 for a given block (jic this has some influence on some config)

---

should we have:

- block gas limit be realistic for the deployed network (keep in mind some other things won't be, such as all contracts being deployed in the same EVM as opposed to different L2s)
- block gas limit be unrealistic, but compensate for the above and allow more calls in the same block, which could point us in the direction of an otherwise-unreachable bug?

---

If we can try to try the tests on a realistic environment, is better for me. Otherwise, if it becomes an issue or is really complex to handle, we can increase it